CHANGELOG.md

```diff
@@ -8,6 +8,39 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 <!-- markdownlint-disable MD024 -->
 <!-- markdownlint-disable MD004 -->
 
+## [Unreleased] (date goes here)
+
+### Migration Tips
+
+If you have a simple program without an action handler, you will now get an error if
+there are missing command-arguments.
+
+```
```

Comment:
I think an indent width of 2 is better.
Because other code examples are 2.

It would be nice to have a highlight setting as well. ` ```js `

Reply:
👍